static/app/views/insights/common/components/chart.tsx

```diff
@@ -454,16 +454,10 @@ function Chart({
     if (type === ChartType.BAR) {
       return (
         <BarChart
+          {...zoomRenderProps}
           height={height}
           series={series}
-          xAxis={{
-            type: 'category',
-            axisTick: {show: true},
-            truncate: Infinity, // Show axis labels
-            axisLabel: {
-              interval: 0, // Show _all_ axis labels
-            },
-          }}
+          xAxis={xAxis}
```

Comment:
TODO: Double check that this isn't needed somewhere in an insights module.

Reply:
All other instances of this component seem to be explicitly asking for a line/area chart. So I think we're fine here.